phpstan.neon

```diff
@@ -29,3 +28,4 @@ parameters:
             - JsonException
             - PHPUnit\Framework\Exception
             - ReflectionException
+            - Psr\Cache\InvalidArgumentException
```

Comment:
Shouldn't we catch this in the sf handler and rethrow a generic exception ? (or just forward it and declare it in the contract)

Reply:
PSR's InvalidArgumentException is an interface that does not extend throwable (at least [in its 1.0 version](https://github.com/php-fig/cache/blob/1.0.1/src/InvalidArgumentException.php)), thus cannot be properly catched nor declared as a `@throws` in phpdoc without making phpstan yell.
I'm not sure how to handle this case without dropping support for symfony 3.4's components

Reply:
we can drop support for symfony 3 and 4 since they are not supported anymore

Reply:
Ok, after further investigations, psr/cache ^2.0 [requires at least php 8](https://github.com/php-fig/cache/blob/2.0.0/composer.json#L13)

Reply:
Ok, we should remember to remove this after php 8 update